components/collection/drop/HolderOfGenerative.vue

```diff
@@ -350,21 +397,25 @@ const submitMint = async (sn: string) => {
       name: result.name,
       collectionName: collectionName.value,
     }
+
+    isSuccessModalActive.value = true
   } catch (error) {
     toast($i18n.t('drops.mintPerAddress'))
     isImageFetching.value = false
+    $consola.error(error)
     throw error
   }
 }
 
 const checkAvailableNfts = async () => {
   availableNfts.isLoading = true
-  const nftEntities = holderOfCollectionData.value?.nftEntities || []
-  const nftIds = nftEntities.map((nft) => nft.sn)
+  const nftIds =
+    collectionData.value?.collectionEntity.nfts.map((nft) => nft.sn) || []
   const claimed = await Promise.all(
     nftIds.map((sn) => isNftClaimed(sn, holderOfCollectionId.value as string)),
   )
-  availableNfts.amount = claimed.filter((x) => !x).length
+  availableNfts.amount =
+    maxMintLimitForCurrentUser.value - claimed.filter((x) => !x).length
   availableNfts.isLoading = false
```

Comment:
It is incorrect. The root issue is that `isNftClaimed` method does not work. https://github.com/kodadot/nft-gallery/issues/9404